recipes/nodejs/nodejs_4.inc

```diff
@@ -36,6 +36,8 @@ GYP_DEFINES_append_mipsel = " mips_arch_variant='r1' "
 PACKAGECONFIG[zlib] = "--shared-zlib,,zlib"
 PACKAGECONFIG[openssl] = "--shared-openssl,,openssl"
 
+EXTRA_OECONF_remove = "--disable-static"
+
```

Comment:
I think the better approach would be to do: `DISABLE_STATIC = ""`. I haven't tested it tough.